addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapInjectionScanRule.java

```diff
@@ -138,12 +137,9 @@ public String getReference() {
     @Override
     public void init() {
         // DEBUG: turn on for debugging
-        // log.setLevel(org.apache.log4j.Level.DEBUG);
-        // this.debugEnabled = true;
+        // LoggerContext.getContext().getConfiguration().getLoggerConfig(this.getClass().getName()).setLevel(Level.DEBUG);
```

Comment:
I'd suggest removing it instead.
Note that this configuration change would have to be applied to take effect.